### PR TITLE
Added self-service SVN-Git migration

### PR DESCRIPTION
**Pull request by tknippenberg**
Added planned release for self-service SVN to Git migration

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@tknippenberg please remove from Draft status when ready so @MariaShaposhnikova can review as part of other docs for release

**Comment by MariaShaposhnikova** [comment]
@Adam-Dupaski , I have reviewed the release note.

File: content/en/docs/releasenotes/developer-portal/_index.md [hunk 1]
```diff
@@ -20,8 +20,8 @@ To see the current status of the Mendix Developer Portal and Control Center, see
 ### November 16th, 2022
 
 #### New features
-* We've introduced a self-service option for Teamserver SVN that allows Scrum Masters to permanently migrate eligible apps to Git.
-  * The [migration documentation](https://docs.mendix.com/developerportal/collaborate/migrate-to-git/) contains more information about eligibility and the migration process. As the process of working with Git is slightly different we have documented the [differences here](https://docs.mendix.com/refguide/svn-git-differences).
+
```
Comment (Adam-Dupaski) [inline content/en/docs/releasenotes/developer-portal/_index.md:23]: "features" > "Features"
"Teamserver" > "Team Server"
"see [Migrating from SVN to Git: Steps After Migration and Process Differences]" > "see the [Migrating from SVN to Git: Steps After Migration and Process Differences] section in ___"

Will need a front-page release date update.

@MariaShaposhnikova

Comment (MariaShaposhnikova) [reply content/en/docs/releasenotes/developer-portal/_index.md:23]: Implemented except the last one, this is not a section, but a doc name.

**Review by Adam-Dupaski — APPROVED** [review]

**Pull request merged**